cmd/client-s3.go

```diff
@@ -54,6 +56,19 @@ const (
 	googleHostName = "storage.googleapis.com"
 )
 
+// getNumMultiPartThreads gets the number of multipart threads to be used for mc cp/mirror functions
+func getNumMultiPartThreads() uint {
+	multipartThreadStr := os.Getenv("DEFAULT_MULTIPART_THREADS")
+	if multipartThreadStr != "" {
+		numThreads, err := strconv.ParseUint(multipartThreadStr, 10, 32)
+		if err != nil {
+			log.Fatal("DEFAULT_MULTIPART_THREADS must be a positive integer.")
+		}
+		return uint(numThreads)
+	}
+	return 0
```

Comment:
Set this to 4 as default.

Reply:
I thought the default should be decided by minio-go as opposed to hardcoding the default in mc as well. When the opts has 0, minio-go sdk automatically overrides it to the default of 4

Reply:
mc can have its own default as well, just for clarity. minio-go can have its own behavior we don't need to tie this with `minio-go`

Reply:
done.